serde_macros/Cargo.toml

```diff
@@ -18,15 +18,15 @@ unstable-testing = ["clippy", "serde/unstable-testing", "serde_codegen/unstable-
 
 [dependencies]
 clippy = { version = "^0.*", optional = true }
-serde_codegen = { version = "0.8.0", default-features = false, features = ["unstable"] }
+serde_codegen = { version = "=0.8.0", default-features = false, features = ["unstable"] }
 
 [dev-dependencies]
 clippy = "^0.*"
 compiletest_rs = "^0.2.0"
 fnv = "1.0"
 rustc-serialize = "^0.3.16"
-serde = "0.8.0"
-serde_test = "0.8.0"
+serde = "=0.8.0"
```

Comment:
No I guess not - I just changed every dependency of a serde crate on a different serde crate. We should be okay constraining just serde_macros->serde_codegen and serde_codegen->serde_codegen_internals.